mautic/mautic
# GrapesJs as default builder

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for 4.0
| Bug fix?                               | no
| New feature?                           | yes
| Deprecations?                          | yes
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
This make GrapesJs as default builder during the installation.



#### Steps to test this PR:
1. Make new installation
2. Go and open email
3. GrapesJs should be default builder
